app/controllers/v0/health_care_applications_controller.rb

```diff
@@ -80,6 +81,15 @@ def download_pdf
 
     private
 
+    def validate_for_bgs
+      mpi_data = MPIData.new(current_user.identity)
+      response = mpi_data.response_from_redis_or_service(user_key: 'mpi_verification')
```

Comment:
you don't need to make a call to MPI. you can just check `current_user.loa3?` and `current_user.ssn.present?`